roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -135,6 +147,16 @@
   when:
     - not build_from_sources
 
+- name: Kibana optimization (can take a while)
+  shell: NODE_OPTIONS=" {{ node_options }} " /usr/share/kibana/bin/kibana --optimize
```

Comment:
Here too

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded